web/packages/test/scripts/set-env.sh

```diff
@@ -12,10 +12,19 @@ zombienet_data_dir="$output_dir/zombienet"
 export PATH="$output_bin_dir:$PATH"
 export polkadot_sdk_dir="${POLKADOT_SDK_DIR:-../polkadot-sdk}"
 
+if [ "$is_electra" == "true" ]; then
+    HOST=$(ifconfig | grep -Eo 'inet (addr:)?([0-9]*\.){3}[0-9]*' | grep -Eo '([0-9]*\.){3}[0-9]*' | grep -v '127.0.0.1')
```

Comment:
This may not work for machines with multi-network interfaces, my PC, e.g. 

```
➜  test git:(electra) ✗ ifconfig | grep -Eo 'inet (addr:)?([0-9]*\.){3}[0-9]*' | grep -Eo '([0-9]*\.){3}[0-9]*' | grep -v '127.0.0.1'
192.168.0.127
26.26.26.1
192.168.64.1
```

Reply:
You may be able to force bind docker to 127.0.0.1 when forwarding ports from host to container. But also maybe we should build from source to avoid docker.

Reply:
Build from source: https://github.com/Snowfork/snowbridge/pull/1283/commits/23eb81d5bdf7951a725f2861831c4a5d0f9aa604